### PR TITLE
Bump libocr-20201217004406-38540d6302ee

### PR DESCRIPTION
**Pull request by kaleofduty**

### CODE REVIEW

File: core/services/pipeline/runner_test.go [hunk 1]
```diff
@@ -599,9 +599,8 @@ ds1 -> ds1_parse;
 		var se []models.JobSpecErrorV2
 		err = db.Find(&se).Error
 		require.NoError(t, err)
-		require.Len(t, se, 2)
+		require.Len(t, se, 1)
```
Comment (samsondav) [inline core/services/pipeline/runner_test.go:602]: Huh, why did this change?

**Review by topliceanu — APPROVED** [review]

File: core/services/pipeline/runner_test.go (hunk 1 above)
Comment (kaleofduty) [reply core/services/pipeline/runner_test.go:602]: Tests fail without it

Comment (kaleofduty) [reply core/services/pipeline/runner_test.go:602]: Because we now no longer falsely log context shutdowns as timeout errors under certain circumstances

Comment (samsondav) [reply core/services/pipeline/runner_test.go:602]: Gotcha, awesome

**Review by samsondav — APPROVED** [review]

**Pull request merged**